### PR TITLE
✨(project) add core workspace

### PR DESCRIPTION
**Pull request by jmaupetit**
## Purpose

Working on grafana dashboards requires to configure a tool suite that can handle Jsonnet sources to provision the local instance of grafana.

## Proposal

Current work implements a developer-friendly working environment using Docker. It includes:

- a configured and provisioned grafana service
- `jb`, the jsonnet package manager
- the `jsonnet` compiler
- an example dashboard

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
+version: "3.5"
+
+services:
+  postgresql:
+    image: postgres:9.6
```
Comment (lunika) [inline docker-compose.yml:5]: you can't use a more recent image ?

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+DOCKER_USER          = $(DOCKER_UID):$(DOCKER_GID)
+COMPOSE              = DOCKER_USER=$(DOCKER_USER) docker-compose
+COMPOSE_RUN          = $(COMPOSE) run --rm
+
+# -- Utils
+WAIT_DB              = $(COMPOSE_RUN) dockerize -wait tcp://postgresql:5432 -timeout 60s
+WAIT_GRAFANA         = $(COMPOSE_RUN) dockerize -wait http://grafana:3000 -timeout 60s
+
+# -- Targets
+#  FIXME
```
[the remark this answers is not in the document]
Comment (jmaupetit) [reply Makefile:14]: None. Should remove this. Well spotted.

File: docker-compose.yml (hunk 1 above)
Comment (jmaupetit) [reply docker-compose.yml:5]: Sure I can. I choose this version used in other FUN projects. What version do you suggest?

Comment (lunika) [reply docker-compose.yml:5]: The more recent grafana can use ?

File: env.d/grafana [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Database
+GF_DATABASE_TYPE=postgres
+GF_DATABASE_HOST=postgresql:5432
+GF_DATABASE_NAME=potsie
+GF_DATABASE_USER=fun
+GF_DATABASE_PASSWORD=pass
+
+# Auth
+GF_SECURITY_ADMIN_PASSWORD="pass"
```
Comment (jmaupetit) [inline env.d/grafana:9]:
```suggestion
GF_SECURITY_ADMIN_PASSWORD=pass
```

**Review by quitterie-lcs — APPROVED** [review]

File: docker-compose.yml (hunk 1 above)
Comment (jmaupetit) [reply docker-compose.yml:5]: Okay I've switched to `13.3`

**Pull request merged**